types/geojson/index.d.ts

```diff
@@ -1,123 +1,157 @@
-// Type definitions for GeoJSON Format Specification Revision 1.0
-// Project: http://geojson.org/
+// Type definitions for geojson 7946.0
```

Comment:
The version number is fine, it's based on the new RFC 7946 GeoJSON Specification.

Reply:
`geojson` on npm is on version 0.5, is that unrelated to this?

Reply:
`geojson` on npm is not how people are using `@types/geojson`.

IMO, the most popular libraries that are dependent on `@types/geojson` are:

- [`@turf/turf`](https://github.com/Turfjs/turf/blob/master/packages/turf-helpers/lib/geojson.d.ts) - Currently embeded `@types/geojson` since there was some breaking change to the repo due to the last PR merge, however once the TurfJS types are stable again we will be moving towards `@types/geojson` again.
- [shapefile](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/shapefile/index.d.ts)
- [leaflet](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/leaflet/index.d.ts)
- [mapboxgl](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mapbox-gl/index.d.ts)
- And more...

@andy-ms